tools/docker/generate_dockerfiles.py

```diff
@@ -423,15 +420,23 @@ def install_cp2k_cmake() -> str:
 RUN /bin/bash -c " \
     echo 'Compiling cp2k...' && \
     source /opt/cp2k-toolchain/install/setup && \
-    cmake -Werror=dev -DCP2K_USE_VORI=ON -DCP2K_USE_COSMA=NO -DCP2K_USE_LIBXSMM=ON -DCP2K_BLAS_VENDOR=OpenBLAS -DCP2K_USE_SPGLIB=ON -DCP2K_USE_LIBINT2=ON -DCP2K_USE_LIBXC=ON -DCP2K_USE_LIBTORCH=OFF .. |& tee ./cmake.log && \
+    cmake -Werror=dev -DCP2K_USE_VORI=ON -DCP2K_USE_COSMA=NO -DCP2K_BLAS_VENDOR=OpenBLAS -DCP2K_USE_SPGLIB=ON -DCP2K_USE_LIBINT2=ON -DCP2K_USE_LIBXC=ON -DCP2K_USE_LIBTORCH=OFF -DCP2K_ENABLE_CONSISTENCY_CHECKS=ON -DCP2K_BUILD_DBCSR=ON -DCP2K_ENABLE_REGTESTS=ON .. |& tee ./cmake.log && \
     ! grep -A5 'CMake Warning' ./cmake.log && \
     make -j"
+WORKDIR /opt/cp2k
 COPY ./data ./data
 COPY ./tests ./tests
 COPY ./tools/regtesting ./tools/regtesting
 
 RUN echo "\nSummary: Compilation works fine.\nStatus: OK\n"
 
+# Run regression tests.
+ARG TESTOPTS=""
+
+# Output the report if the image is old and was therefore pulled from the build cache.
+# CMD cat $(find ./report.log -mmin +10) | sed '/^Summary:/ s/$/ (cached)/'
+ENTRYPOINT []
+
```

Comment:
This confuses the CI. I'd suggest we enable the regtests in a follow up PR as this one is already quite large. I could take care of it.